packages/types/src/lo-hi.ts

```diff
@@ -76,7 +76,7 @@ export const fromBaseUnit = (lo = 0n, hi = 0n, exponent: number): BigNumber => {
  * @param {number} exponent - The exponent to be applied.
  * @returns {LoHi} An object with properties `lo` and `hi`, representing the low and high 64 bits of the multiplied value.
  */
-export const toBaseUnit = (value: BigNumber, exponent: number): LoHi => {
+export const toBaseUnit = (value: BigNumber, exponent = 0): LoHi => {
```

Comment:
Since we're occasionally passing in an empty `DenomMetadata` upstream of this call, I'll leave the default exponent as `0`, so that we're just multiplying by `1`.